Sources/SwiftSyntaxBuilder/IdentifierPatternConvenienceInitializers.swift

```diff
@@ -21,3 +21,13 @@ extension IdentifierPattern: ExpressibleByStringLiteral {
     self.init(value)
   }
 }
+
+extension String: ExpressibleAsIdentifierPattern {
+  public func createPatternBuildable() -> PatternBuildable {
```

Comment:
Should this go into a separate extension? `ExpressibleAsIdentifierPattern` only requires `createIdentifierPattern`.

Reply:
> Should this go into a separate extension?

Do you mean file or do you mean that `public func createPatternBuildable() -> PatternBuildable` should be in another `extension String: ...` ? 

The reason I placed it there is that `ExpressibleAsIdentifierPattern` inherence from `ExpressibleAsPatternBuildable`. 

I could make something like 

```swift
extension {
  public func createPatternBuildable() -> PatternBuildable {
    IdentifierPattern(self)
  }
```

Reply:
I think in that case we should have 

```
extension ExpressibleAsIdentifierPattern {
  public func createPatternBuildable() -> PatternBuildable {
    return self
  }
}
```

Or doesn’t that work? (I haven’t checked)

Reply:
PR in #335